.github/workflows/CI.yaml

```diff
@@ -54,6 +55,10 @@ jobs:
           components: clippy, rustfmt
           target: wasm32-unknown-unknown
 
+      - name: "Install: Show specific nightly version"
+        if: ${{ env.NIGHTLY_TOOLCHAIN_VERSION != '' }}
+        run: echo $NIGHTLY_TOOLCHAIN_VERSION | sed 's/-/ - /g'
```

Comment:
What is this for?

Reply:
to be able to check what version was pinned